src/main/java/com/useful/ucars/MotionManager.java

```diff
@@ -111,6 +111,12 @@ public static void move(Player player, float f, float s, boolean jumping) { // '
 		}
 		
 		boolean inAir = car.getLocation().clone().add(0, -1, 0).getBlock().isEmpty();
+		if(!car.getLocation().clone().add(0.5, -1, 0).getBlock().isEmpty() || !car.getLocation().clone().add(-0.5, -1, 0).getBlock().isEmpty() ||
+				!car.getLocation().clone().add(0, -1, 0.5).getBlock().isEmpty() || !car.getLocation().clone().add(0, -1, -0.5).getBlock().isEmpty() ||
+				!car.getLocation().clone().add(0.5, -1, 0.5).getBlock().isEmpty() || !car.getLocation().clone().add(0.5, -1, -0.5).getBlock().isEmpty() ||
+				!car.getLocation().clone().add(-0.5, -1, 0.5).getBlock().isEmpty() || !car.getLocation().clone().add(-0.5, -1, -0.5).getBlock().isEmpty()) { //Check area under car
```

Comment:
Maybe should be using half the entity width instead of 0.5?

Reply:
Good point, done - commit coming soon